Gemfile

```diff
@@ -1,12 +1,13 @@
 source "https://rubygems.org"
 
-# Specify your gem's dependencies in codeclimate-test-reporter.gemspec
-gemspec
+gem "rake"
+gem "rspec"
+gem "webmock"
+gem "pry"
```

Comment:
Sorry, I didn't see this thread before commenting: https://github.com/codeclimate/ruby-test-reporter/commit/a0e5d038dfbcaed56e5b7b2b30a7ec2ab1c468db#commitcomment-19735625

Still thinking about how to manage this...